### DISCUSSION TITLE
SyntaxError: Cannot use import statement outside a module

### DISCUSSION THREAD

**Discussion by crrojas** [mercadopago/sdk-react#85] [Q&A] [answered]
Hola,

Estoy tratando de integrar mercado pago en nextjs 12.2 y al ejecutarla siguiente línea de código que está en la página oficial con la documentación

import { initMercadoPago, Wallet } from '@mercadopago/sdk-react';

me lanza el siguiente eror

Server Error
SyntaxError: Cannot use import statement outside a module

¿Alguien sabe cómo arreglarlo?

Desde ya gracias,

**Comment by francineemilia** [comment, marked as answer]
Hola @crrojas 

En esta [discusión](https://github.com/mercadopago/sdk-react/discussions/71) tenemos un caso similar con una explicación muy completa, pero se tiene alguna dificuldad con portugués y compreendes mejor inglés, tenemos un [artigo em nuestra wiki](https://github.com/mercadopago/sdk-react/wiki/Using-Bricks-with-Next.js) com una explicación semelhante.

Se necesitas de más alguno apoyo, avisenos
